packages/@adobe/react-spectrum/package.json

```diff
@@ -91,6 +92,7 @@
     "@react-spectrum/theme-dark": "^3.5.15",
     "@react-spectrum/theme-default": "^3.5.15",
     "@react-spectrum/theme-light": "^3.4.15",
+    "@react-spectrum/toast": "3.0.0-beta.18",
     "@react-spectrum/tooltip": "^3.7.1",
     "@react-spectrum/tree": "^3.0.0-beta.3",
```

Comment:
```suggestion
    "@react-spectrum/tree": "3.0.0-beta.3",
```

Reply:
it'll get overwritten when the versions are bumped so this is unnecessary